### ISSUE TITLE
The frontend should list which files have been uploaded/validated after validation

### ISSUE THREAD

**Issue by SLornieCYC** [SocialFinanceDigitalLabs/quality-lac-data-beta#62]
Connected to #61.  After you choose your files and click Validate there is nothing that tells the user _which_ files have been loaded.

As each rule depends on the existence of a specific file or combination of files it is important to show this information on-screen as a reminder of what has been loaded in this session.  This could be as simple as a comma-separated list of submitted files (actual file name or the internal file type e.g. "Episodes", "Reviews"), or a more comprehensive feature that shows all of the "expected" files (current and previous year) and indicates which ones have and haven't been provided.

**Comment by dezog** [comment]
Good idea, maybe this could be combined with the filter idea: de-/select by table name, with those that haven't been uploaded greyed out or in red or similar?
